src/sentry/static/sentry/app/actionCreators/environments.jsx

```diff
@@ -1,4 +1,4 @@
-import OrganizationEnvironmentsStore from 'app/stores/organizationEnvironmentsStore';
+import OrganizationEnvironmentActions from 'app/actions/environmentActions';
```

Comment:
Should this be `EnvironmentActions` to match the filename?

Reply:
Yea was unsure about a good name. I didn't want to make it seem like `fetchOrganizationEnvironments` was calling actions from a separate set of actions/store (`EnvironmentActions` as opposed to `OrganizationEnvironmentActions`) but it's probably better to match the names or else it may be even more confusing.

Reply:
actionCreators doesn't need to be 1:1 to a set of actions (and same with actions <--> stores)